source/game/field/obj/ObjectWoodbox.cc

```diff
@@ -0,0 +1,27 @@
+#include "ObjectWoodbox.hh"
+
+#include "game/field/CollisionDirector.hh"
+
+namespace Field {
+
+/// @addr{0x8077E5E4}
+ObjectWoodbox::ObjectWoodbox(const System::MapdataGeoObj &params) : ObjectBreakable(params) {}
+
+/// @addr{0x8077E620}
+ObjectWoodbox::~ObjectWoodbox() = default;
+
+/// @addr{0x8077EBB8}
+void ObjectWoodbox::calcCollisionTransform() {
+    constexpr f32 HALF_SIZE = 100.0f;
```

Comment:
I forgot we can do this
```suggestion
    constexpr f32 HALF_SIZE = 100.0f;
    constexpr EGG::Vector3f POS_OFFSET = EGG::Vector3f(0.0f, HALF_SIZE, 0.0f);
```